manifests/k/karakun/OpenWebStart/1.10.1/karakun.OpenWebStart.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: karakun.OpenWebStart
+PackageVersion: 1.10.1
+InstallerLocale: en-US
+InstallerType: portable

```

Comment:
Both _Komac_ version `2.2.1` and `2.3.0` regard this package as `portable` instead of `exe`. 

- This cause issue #159914.
- The related issue will be resolved by #159915.

Reply:
CC @RussellBanks